pandas/core/internals/construction.py

```diff
@@ -65,8 +67,6 @@
 if TYPE_CHECKING:
     from numpy.ma.mrecords import MaskedRecords
 
-    from pandas import Series
```

Comment:
Curious what happened here. Series is imported inline below, so am I correct to say that this is sufficient for mypy?

Reply:
Thanks!

And yes, seems like it, e.g.:
`t.py`:
```python
def foo() -> None:
    a: List[int]
    reveal_type(a)
    from typing import List
```

```console
$ python3.8 -m mypy t.py 
t.py:3: note: Revealed type is 'builtins.list[builtins.int]'
```